### PR TITLE
Coverage for 'gcloud._apitools.exceptions'

### PR DESCRIPTION
**Pull request by tseaver**
~~Uses #1190 as a base.~~

### CODE REVIEW

**Comment by tseaver** [comment]
@dhermes Rebased after merge of #1190.

File: gcloud/_apitools/test_exceptions.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+        self.assertEqual(exception.response, RESPONSE)
+        self.assertEqual(exception.content, CONTENT)
+        self.assertEqual(exception.url, URL)
+        self.assertEqual(exception.status_code, 404)
+        self.assertEqual(
+            str(exception),
+            "HttpError accessing <http://www.example.com>: "
+            "response: <{'status': '404'}>, content <CONTENT>")
+
+    def test_FromResponse(self):
```
Comment (dhermes) [inline gcloud/_apitools/test_exceptions.py]: When do you plan to change method names (`FromResponse` here) to adhere to PEP8?

**Comment by dhermes** [comment]
LGTM

File: gcloud/_apitools/test_exceptions.py (hunk 1 above)
Comment (tseaver) [inline gcloud/_apitools/test_exceptions.py]: I'll do cleanups / refactoring after getting 100% coverage.

**Pull request merged**